src/aspire/abinitio/commonline_c3_c4.py

```diff
@@ -373,7 +373,9 @@ def _estimate_inplane_rotations(self, vis):
             # Q is a rank-1 Hermitian matrix.
             eig_vals, eig_vecs = eigh(Q)
             leading_eig_vec = eig_vecs[:, -1]
-            logger.info(f"Top 5 eigenvalues of Q are {str(eig_vals[-5:][::-1])}.")
+            logger.info(
+                f"Top 3 eigenvalues of Q (rank-1) are {str(eig_vals[-3:][::-1])}."
```

Comment:
Don't we want to see the eigen spacing here?  I see it says rank 1 so maybe 3 is already enough. 

Was this a legacy diagnostic message we brought over?

Reply:
I changed it to 3 because I didn't want it to be confused with the eigenvalue shelf we look for with CLSyncVoting. Also why I added "rank-1", ie. to indicate that we're looking for one large eigenvalue.

Yes, this legacy log message.

Reply:
If you have no reason to believe the 5 values was intentional then I am fine with it.  Thanks!